internal/scaffold/cmd.go

```diff
@@ -217,8 +217,12 @@ func addMetrics(ctx context.Context, cfg *rest.Config, namespace string) {
 // serveCRMetrics gets the Operator/CustomResource GVKs and generates metrics based on those types.
 // It serves those metrics on "http://metricsHost:operatorMetricsPort".
 func serveCRMetrics(cfg *rest.Config) error {
-	// Below function returns filtered operator/CustomResource specific GVKs.
-	// For more control override the below GVK list with your own custom logic.
+	// The function below returns filtered operator/CR specific GVKs. For more control, override the GVK list below
```

Comment:
```suggestion
	// The function below returns a list of filtered operator-/CR-specific GVKs.
	// To collect metrics on fewer types, filter the GVK list below further.
```

Reply:
I'd rather see this wording than the wording you're currently using. There is no "overriding" here, just further filtering.